packages/prisma-next/sync-server/README.md

```
@@ -94,22 +94,65 @@ Step 2 exists because a stamped `scopeKey` is a snapshot, and ownership can move
+  orm: defineConfig({
+    schema: "src/lib/prisma/schema.prisma",
+    db: { connection: process.env.DATABASE_URL },
+  }),
+});
+```
+
+`createSyncServer`'s `contract` can then be this real Postgres contract directly (with a `getKeyField` override — see the API section above) — no IDB-shaped stand-in needed just to feed the DAG.
+
+**Upgrading from a hand-authored `Changelog`:** delete the `Changelog` model and `ChangeOperation` enum from `schema.prisma` before switching to `sqlContractWithSync`/this facade. Both are appended for you — leaving your own declarations in place produces duplicate PSL declarations, which fail contract generation.
+
+#### Other targets, or composing your own config
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)